Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/validation/core/business/wrapper/MappingsTest.kt

```diff
@@ -49,64 +49,64 @@ class MappingsTest : BaseTest() {
     @Test
     fun `filter rules works`() {
         val validationClock = Instant.parse("2021-05-27T07:46:40Z")
-        val validationDate = validationClock.toDateTime(DateTimeZone.UTC)
+        val validationDate = validationClock.atZone(UTC_ZONE_ID)
 
         val vacA1 = createDccRule(
             certificateType = RuleCertificateType.VACCINATION,
             identifier = "VR-DE-1",
             version = "1.0.0",
-            validFrom = validationClock.minus(100).toString(),
-            validTo = validationClock.plus(100).toString(),
+            validFrom = validationClock.minus(100, ChronoUnit.DAYS).toString(),
```

Comment:
this used to be milliseconds to catch time conversion errors, with days it is too broad